spec/fixtures/worker_basic_suite.rb

```diff
@@ -0,0 +1,4 @@
+[
+  {:workers => 2, :name => "simple 1", :config => "simple.conf", :input => "simple_10.txt", :time => 5},
```

Comment:
the same questions as earlier apply here,

* Why having always the workers parameter? 
* Why 2 workers?

Thanks,